requirements.txt

```diff
@@ -1,5 +1,5 @@
 PyQt5==5.12.3
```

Comment:
PyQt5==5.15.9 in https://github.com/AllYarnsAreBeautiful/ayab-desktop/pull/572 ?

Reply:
Is there any way to get away from the PyQt5 dependency? It would be nice not to have dependencies on both PyQt5 and PySide6.